Create 06-counter-storage.md (#535)
cyclr/docs: pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md

---
title: Counter Storage
sidebar: cyclr_sidebar
permalink: counter-storage
tags: [utility-connector]
menus:
    data-storage:
        title: Counter Storage
        identifier: counter-storage
        toggleonly: menutoggleonly
        weight: 2
---
{::options parse_block_html="true" /}
<section class="card">
  
> **_PLEASE NOTE:_** Using Utility Connectors counts toward your overall task usage.

</section>
<section class="card">
  
## Purpose

> To keep multiple counters, scoped globally or to a Cycle, you can use the **Counter Storage** Connector.

Reviewer: ```suggestion
 The **Counter Storage** Connector allows for multiple counters, either stored globally or within a Cyclr, to be tracked.
```

> Counters can be created and updated in one part of the Cycle and retrieved in another.

Reviewer: ```suggestion
Counters can be created and/or updated in one step of a Cycle and retrieved from another step in a Cycle.
```

Cycle Counters are only be accessible _within the same cycle_, where as Global Cycle Counters _persist across your account_.

> The **Counter Storage** Connector works by using a Name and an Amount. The Name must be unique. The Amount will be 1 by default and can accept float values.

Reviewer: ```suggestion
The **Counter Storage** Connector stores and/or adjusts an Amount against a unique Counter Name. The Amount will be adjusted by 1 by default and can accept float values.
```
